### ISSUE TITLE
train T2T-ViT on the  custom dataset

### ISSUE THREAD

**Issue by 123-xin** [yitu-opensource/T2T-ViT#55]
Hello, thank you very much for your work. 
Can you tell me how my training set and label format should be set up if I train the network on a custom dataset.
(My custom dataset is divided into two categories )

**Comment by yuanli2333** [comment]
Hi, you should write a dataloader to load your own data,

Or if you want to use our repo to train, you should reorganize the data structure of your dataset as ImageNet:

│imagenet/
├──train/
│  ├── n01440764
│  │   ├── n01440764_10026.JPEG
│  │   ├── n01440764_10027.JPEG
│  │   ├── ......
│  ├── ......
├──val/
│  ├── n01440764
│  │   ├── ILSVRC2012_val_00000293.JPEG
│  │   ├── ILSVRC2012_val_00002138.JPEG
│  │   ├── ......
│  ├── ......
